docs/themes/avocadocs/layouts/_default/_markup/render-link.html

```diff
@@ -4,7 +4,7 @@
   {{- $isRemote := in $remoteSchemes $url.Scheme -}}
   {{- $isFile := strings.HasSuffix .Destination "pdf" }}
   {{- if or $isRemote $isFile -}}
-    <a href="{{ .Destination }}" target="_blank" rel="noopener">{{ .Text | safeHTML }}</a>
+    <a href="{{ htmlUnescape .Destination }}" target="_blank" rel="noopener">{{ .Text | safeHTML }}</a>
```

Comment:
:question: What is this file/part doing?

Reply:
The link gets passed with an ampersand (`&`). This ensures it doesn't get encoded as `&amp;`, as it does otherwise.